models/database_test.go

```diff
@@ -381,4 +384,35 @@ func TestDatabaseMigrations(t *testing.T) {
 			LR: 60 * time.Second,
 		}, settings.MetricsResolutions)
 	})
+	t.Run("Update type for stats_collections field", func(t *testing.T) {
```

Comment:
Can we add one more test for upgrade from 2.26.0 to 2.27.0?

Reply:
Done